### PR TITLE
Sync v2-8-stable with v2-8-test to release 2.8.1

### PR DESCRIPTION
**Pull request by ephraimbuddy**
Time for `2.8.1rc1`!

### CODE REVIEW

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -21,22 +21,174 @@
...
+- Explicit string cast required to force integer-type run_ids to be passed as strings instead of integers (#36756)
+- Add log lookup exception for empty ``op`` subtypes (#35536)
+- Remove usused index on task instance (#36737)
+- Fix check on subclass for ``typing.Union`` in ``_infer_multiple_outputs`` for Python 3.10+ (#36728)
+- Make sure ``multiple_outputs`` is inferred correctly even when using ``TypedDict`` (#36652)
+- Add back FAB constant in legacy security manager (#36719)
+- Fix AttributeError when using ``Dagrun.update_state`` (#36712)
+- Do not let ``EventsTimetable`` schedule past events if ``catchup=False`` (#36134)
+- Support encryption for triggers parameters (#36492)
+- Rename concurrency label to max active tasks (#36691)
```
Comment (eladkal) [inline RELEASE_NOTES.rst]: I would move this to misc. It's not a bug fix just renaming item in the UI

**Review by eladkal — APPROVED** [review]
LGTM

**Review by hussein-awala — APPROVED** [review]
LGTM

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
BTW. the #36792 would not be effective until merged :)

**Comment by ephraimbuddy** [comment]
> BTW. the #36792 would not be effective until merged :)

Oh, yeah, that's true

**Comment by potiuk** [comment]
> Oh, yeah, that's true

But will be working for 2.8.2 :)

**Pull request merged**